src/org/traccar/helper/Checksum.java

```diff
@@ -243,4 +243,12 @@ public static long luhn(long imei) {
         return (10 - (checksum % 10)) % 10;
     }
 
+    public static int modulo256(byte[] bytes) {
```

Comment:
`modulo 256 checksum` is official name of this alghoritm. But I can rename this one, if you really want it.